app/services/event_generator/geoserver_event_generator.rb

```diff
@@ -14,8 +14,14 @@ def derivatives_created(record)
     end
 
     def derivatives_deleted(record)
+      # Attempt to delete from both public and restricted
+      # workspaces to make sure all traces of the file
+      # are cleaned up on GeoServer.
       publish_message(
-        message("DELETED", record)
+        message("DELETED", record, public_workspace)
+      )
+      publish_message(
```

Comment:
💯 To this approach, I definitely could see how changing `#authenticated_workspace`  or `Figgy.config["geoserver"]["authenticated"]["workspace"]` in the config. would provide adopters with the ability to have separate GeoServer instances if needed